lib/public/Model.js

```diff
@@ -269,6 +269,15 @@ export default class Model extends Observable {
         this.notify();
     }
 
+    /**
+     * Toggle navbar LHC dropdown
+     * @returns {undefined}
+     */
+    toggleLHCDropdown() {
```

Comment:
I think here you can extract a common toggle function, to which you simply pass a key

Reply:
refactored